src/app/redux/UserSaga.js

```diff
@@ -82,11 +84,49 @@ export const userWatches = [
 
 const highSecurityPages = [
     /\/market/,
-    /\/@.+\/(transfers|permissions|password|communities)/,
+    /\/@.+\/(transfers|permissions|password|communities|delegations)/,
     /\/~witnesses/,
     /\/proposals/,
 ];
 
+function* getVestingDelegationsSaga(action) {
+    try {
+        yield call(
+            [api, api.getVestingDelegations],
+            action.payload.account,
+            '',
+            1000,
+            action.payload.successCallback
+        );
+    } catch (error) {
+        console.log(error);
+    }
+}
+
+function* getExpiringVestingDelegationsSaga(action) {
+    try {
+        let result = yield call(fetch, 'https://api.steemit.com', {
```

Comment:
The previous DIP adds a node selection, can we use that instead of hard-coded the Node.

Reply:
@VictorVonFrankenstein  

This is the recommend way: 

```
function* getExpiringVestingDelegationsSaga(action) {
    try {
        yield call(
            [api, api.send],
            'database_api',
            {
                method: 'find_vesting_delegation_expirations',
                params: {
                    account: action.payload.account,
                },
            },
            action.payload.successCallback
        );
    } catch (error) {
        console.log(error);
    }
}
```

Reply:
Change pushed.